packages/valory/skills/hello_world_abci/rounds.py

```diff
@@ -194,6 +224,8 @@ class HelloWorldAbciApp(AbciApp[Event]):
     """
 
     initial_round_cls: AppState = RegistrationRound
+    # TODO: Question: Why is this defined here and also in fsm_specification.yaml?
```

Comment:
AbciApp it's just the transition function.
The file `fsm_specification.yaml` is used to check the python implementation against a "simple" specification format. It is also used to scaffold the Abci skill from that specification file.